examples/demo-wdl-project/agc-project.yaml

```diff
@@ -32,6 +32,9 @@ contexts:
     engines:
       - type: wdl
         engine: miniwdl
+        resourceRequirements:
```

Comment:
Are engine `resourceRequirements` a required property now? If they are not then we probably don't need them in the example `agc.project-yaml` files unless we know these values are needed to run the example workflows

Reply:
these are not a required property, I remove them from the example project files.